src/Lykke.SettingsReader/SettingsProcessor/CheckFieldResult.cs

```diff
@@ -40,20 +41,35 @@ public static CheckFieldResult Ok(string propertyName, string url)
         /// </summary>
         /// <param name="propertyName">Property name</param>
         /// <param name="url">Dependency url</param>
-        /// <param name="throwExceptionOnFail">Throw exception flag</param>
-        public static CheckFieldResult Failed(string propertyName, string url, bool throwExceptionOnFail)
+        public static CheckFieldResult Failed(string propertyName, string url)
         {
             return new CheckFieldResult
             {
                 Url = url,
                 Result = false,
-                PropertyName = propertyName,
-                ThrowExceptionOnFail = throwExceptionOnFail,
+                PropertyName = propertyName
             };
         }
+        
+        /// <summary>
+        /// Helper method to init failed result
+        /// </summary>
+        /// <param name="propertyName">Property name</param>
+        /// <param name="fieldValue">Field value</param>
+        /// <param name="message">Error message</param>
+        public static CheckFieldResult Failed(string propertyName, string fieldValue, string message)
+        {
+            return new CheckFieldResult(propertyName, fieldValue, message);
+        }
 
         private CheckFieldResult()
         {
         }
+
+        private CheckFieldResult(string propertyName, string fieldValue, string message)
```

Comment:
Sanitize the fieldValue value. For example a connection string to Azure table must not appear in the log output because it contains a security key.

Reply:
no need to sanitize fieldValue here, Azure checker uses the other method to return failed result and return base url only. This method is used by TcpChecker and returns port value or address